src/Symfony/Bundle/SecurityBundle/Command/UserPasswordEncoderCommand.php

```diff
@@ -94,11 +95,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $encoder = $this->getContainer()->get('security.encoder_factory')->getEncoder($userClass);
         $bcryptWithoutEmptySalt = !$emptySalt && $encoder instanceof BCryptPasswordEncoder;
+        $argon2iWithoutEmptySalt = !$emptySalt && $encoder instanceof Argon2iPasswordEncoder;
```

Comment:
let's do it separately indeed. But the introduction of this encoder makes it worth adding such feature.